.gitlab/tests/contrib.yml

```diff
@@ -1,3 +1,12 @@
+aiohttp:
```

Comment:
All integration changes EXCEPT for opentelemetry here are just reshuffling for alphabetizing the order.